### PR TITLE
 #5608 move the Doc of Ambari Integration to official website repository

### PR DESCRIPTION
**Pull request by kevinfujiang**

### CODE REVIEW

**Comment by CalvinKirs** [comment]
@QuakeWang  please take a look,thx

File: docs/en-us/1.3.6/user_doc/ambari-integration.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+### Instructions for using the Dolphin Scheduler's Ambari plug-in
+
+#### Note
+
+1. This document is intended for users with a basic understanding of Ambari
+2. This document is a description of adding the Dolphin Scheduler service to the installed Ambari service
```
Comment (QuakeWang) [inline docs/en-us/1.3.6/user_doc/ambari-integration.md:6]:
```suggestion
2. This document is a description of adding the DolphinScheduler service to the installed Ambari service
```
Please delete the extra space.

File: docs/en-us/1.3.6/user_doc/ambari-integration.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+   - Manual installation (recommended）：
+      - Copy the prepared RPM packages to each node of the cluster.
+      - Execute with DS installation user: ```rpm -ivh apache-dolphinscheduler-xxx.noarch.rpm```
+      - Mysql-connector-java packaged using the default POM file will not be included.
+      - The RPM package was packaged in the project with the installation path of /opt/soft. 
+        If you use mysql as the database, you need add it manually.
+      
+   - Automatic installation with ambari
+      - Each node of the cluster needs to configure the local yum source
```
Comment (QuakeWang) [inline docs/en-us/1.3.6/user_doc/ambari-integration.md:27]:
```suggestion
      - Each node of the cluster needs to be configured the local yum source
```
The passive voice is recommended

File: docs/en-us/1.3.6/user_doc/ambari-integration.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+4. Copy plug-in directory
+
+   - copy directory ambari_plugin/common-services/DOLPHIN to ambari-server/resources/common-services/
+   - copy directory ambari_plugin/statcks/DOLPHIN to ambari-server/resources/stacks/HDP/2.6/services/--stack version is selected based on the actual situation
+
+5. Initializes the database information
+
+   ```
+   -- Create the database for the Dolphin Scheduler：dolphinscheduler
```
Comment (QuakeWang) [inline docs/en-us/1.3.6/user_doc/ambari-integration.md:38]:
```suggestion
   -- Create the database for the DolphinScheduler：dolphinscheduler
```
As above, delete the extra spaces.

File: docs/en-us/1.3.6/user_doc/ambari-integration.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+   '{password}';
+   flush privileges;
+   ```
+
+ 
+
+#### Ambari Install Dolphin Scheduler
+- **NOTE: You have to install zookeeper first**
+
+1. Install  Dolphin Scheduler on ambari web interface
```
Comment (QuakeWang) [inline docs/en-us/1.3.6/user_doc/ambari-integration.md:55]: As above, delete the extra spaces.

File: docs/en-us/1.3.6/user_doc/ambari-integration.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+   ![](https://dolphinscheduler.apache.org/img/ambari-plugin/DS2_AMBARI_001.png)
+
+2. Select the nodes for the Dolphin Scheduler's Master installation
```
Comment (QuakeWang) [inline docs/en-us/1.3.6/user_doc/ambari-integration.md:59]: As above, delete the extra spaces.

File: docs/en-us/1.3.6/user_doc/ambari-integration.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+   ![](https://dolphinscheduler.apache.org/img/ambari-plugin/DS2_AMBARI_002.png)
+
+3. Configure the Dolphin Scheduler's nodes for Worker, Api, Logger, Alert installation
```
Comment (QuakeWang) [inline docs/en-us/1.3.6/user_doc/ambari-integration.md:63]: As above, delete the extra spaces.

File: docs/en-us/1.3.6/user_doc/ambari-integration.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+   ![](https://dolphinscheduler.apache.org/img/ambari-plugin/DS2_AMBARI_003.png)
+
+4. Set the installation users of the Dolphin Scheduler service (created in step 1) and the user groups they belong to
```
Comment (QuakeWang) [inline docs/en-us/1.3.6/user_doc/ambari-integration.md:67]: As above, delete the extra spaces.

**Review by QuakeWang — COMMENTED** [review]
I have completed the review，@kevinfujiang you can make changes as suggested，thx

**Review by dailidong — APPROVED** [review]
+1
good job

File: docs/en-us/1.3.6/user_doc/ambari-integration.md (hunk 4 above)
Comment (dailidong) [reply docs/en-us/1.3.6/user_doc/ambari-integration.md]: good job, I have made some changes, ^_^

**Pull request merged**